### PR TITLE
Created the templates #52

### LINKED ISSUE #52: Create Templates for Issues and Pull Requests
## Description
Currently we have no standard template for making a Issue and a Pull Request...
To make sure the everything is clear and consistent we should make a GitHub template for Issues and Pull Requests.

### Tasks
- [ ] Add a issue template
- [ ] Add a pull request template

### D.O.D
When you create a Issue and Pull Request we want to see a template to load in when you create an issue!

### PR DESCRIPTION
**Pull request by vQFire**
closes #52 

**What is it supposed to do**
It makes sure we all get some structure in our issues en pull requests

**How can we test your branch**
It is hard to test what you can do is look at the template and check for mistakes in grammar. If you really want to test the templates you need to test it in your own repo. I did test it in my own repo to make sure it works.

**What browser have been checked**
It can't be checked in a browser because it is not in Seaqull.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or designs about the feature request here.
+
+**D.O.D.**
+Describe when your feature would be complete!
```
Comment (AjobK) [inline .github/ISSUE_TEMPLATE/feature.md]: will be

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+closes #XXX
+
+**What is it supposed to do**
+Explain in short what your component is supposed to do
+
+**How can we test your branch**
+Explain how can test your branch Ex. width, height etc...
```
Comment (AjobK) [inline .github/PULL_REQUEST_TEMPLATE.md]: Explain how can?
Ex.?
This is very unclear...
Please do it in the following way

'Preperation before testing (Eg. width, height)'

File: .github/ISSUE_TEMPLATE/feature.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+name: Feature
+about: Suggest an idea for Seaqull
+title: ''
+labels: ''
```
Comment (AjobK) [inline .github/ISSUE_TEMPLATE/feature.md]: Add label improvement? Maybe? If Possible?

File: .github/ISSUE_TEMPLATE/feature.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```
Comment (AjobK) [inline .github/ISSUE_TEMPLATE/feature.md]: you would

File: .github/ISSUE_TEMPLATE/feature.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```
Comment (AjobK) [inline .github/ISSUE_TEMPLATE/feature.md]: you have

File: .github/ISSUE_TEMPLATE/feature.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+---
+name: Feature
+about: Suggest an idea for Seaqull
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (AjobK) [inline .github/ISSUE_TEMPLATE/feature.md]: Unnecessary 'Please describe'

File: .github/ISSUE_TEMPLATE/feature.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+---
+name: Feature
+about: Suggest an idea for Seaqull
+title: ''
```
Comment (AjobK) [inline .github/ISSUE_TEMPLATE/feature.md]: Add 'Issue name | Closes #XXX'

File: .github/ISSUE_TEMPLATE/feature.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or designs about the feature request here.
+
+**D.O.D.**
```
Comment (AjobK) [inline .github/ISSUE_TEMPLATE/feature.md]: Definition of done, fully

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+closes #XXX
+
+**What is it supposed to do**
```
Comment (AjobK) [inline .github/PULL_REQUEST_TEMPLATE.md]: What it is supposed to do

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+closes #XXX
```
Comment (AjobK) [inline .github/PULL_REQUEST_TEMPLATE.md]: Closes #XXX

File: .github/ISSUE_TEMPLATE/feature.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or designs about the feature request here.
```
Comment (AjobK) [inline .github/ISSUE_TEMPLATE/feature.md]: 'about the feature request here' is unnecessary

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+
+**What is it supposed to do**
+Explain in short what your component is supposed to do
+
+**How can we test your branch**
+Explain how can test your branch Ex. width, height etc...
+
+
+**What browser have been checked**
```
Comment (AjobK) [inline .github/PULL_REQUEST_TEMPLATE.md]: Instead of this use

'What browser(s) has/have been checked'
or
'Tested browsers'

**Review by AjobK — CHANGES_REQUESTED** [review]
👍

File: .github/ISSUE_TEMPLATE/feature.md (hunk 7 above)
Comment (vQFire) [reply .github/ISSUE_TEMPLATE/feature.md]: It is hard to close an issue in an issue is it not?

File: .github/ISSUE_TEMPLATE/feature.md (hunk 3 above)
Comment (vQFire) [reply .github/ISSUE_TEMPLATE/feature.md]: This has been updated

File: .github/ISSUE_TEMPLATE/feature.md (hunk 6 above)
Comment (vQFire) [reply .github/ISSUE_TEMPLATE/feature.md]: This has been updated

File: .github/ISSUE_TEMPLATE/feature.md (hunk 4 above)
Comment (vQFire) [reply .github/ISSUE_TEMPLATE/feature.md]: This has been updated

File: .github/ISSUE_TEMPLATE/feature.md (hunk 5 above)
Comment (vQFire) [reply .github/ISSUE_TEMPLATE/feature.md]: This has been updated

File: .github/ISSUE_TEMPLATE/feature.md (hunk 11 above)
Comment (vQFire) [reply .github/ISSUE_TEMPLATE/feature.md]: This has been updated

File: .github/ISSUE_TEMPLATE/feature.md (hunk 8 above)
Comment (vQFire) [reply .github/ISSUE_TEMPLATE/feature.md]: This has been updated

File: .github/ISSUE_TEMPLATE/feature.md (hunk 1 above)
Comment (vQFire) [reply .github/ISSUE_TEMPLATE/feature.md]: This has been updated

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 10 above)
Comment (vQFire) [reply .github/PULL_REQUEST_TEMPLATE.md]: This has been updated

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 9 above)
Comment (vQFire) [reply .github/PULL_REQUEST_TEMPLATE.md]: This has been updated

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (vQFire) [reply .github/PULL_REQUEST_TEMPLATE.md]: This has been updated

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 12 above)
Comment (vQFire) [reply .github/PULL_REQUEST_TEMPLATE.md]: This has been updated

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 13]
```diff
@@ -0,0 +1,16 @@
+Closes #XXX
+
+**What it is supposed to do**
+Explain in short what your component is supposed to do
```
Comment (MiguelFieira) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: Not all issue's are "component" related, I would rather see a Description of what the problem was & what is should do.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+
```
Comment (MiguelFieira) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Maybe add a checklist of tasks in the original issue & put the same one here so we can all see in order what changed

**Review by MiguelFieira — CHANGES_REQUESTED** [review]
Small suggestions...
Dismiss my Review if others agree on the current version

**Review by MiguelFieira — APPROVED** [review]

**Review by Samvdg — APPROVED** [review]

**Pull request merged**